packages/core/ios/LottieReactNative/ContainerView.swift

```diff
@@ -317,6 +309,20 @@ class ContainerView: RCTView {
         }
     }
 
+    func applyContentMode() {
+        guard let animationView else { return }
```

Comment:
This is newer swift syntax, and may break on older versions (We support Swift 5.6) maybe change it to:
```suggestion
        guard let animationView = animationView else { return }
```
To ensure compatibility